MoneySplitter.Infrastructure/IMembershipService.cs

```diff
@@ -9,5 +9,7 @@ public interface IMembershipService
         Task<bool> SingInAndLoadUserDataAsync(string email, string password);
         Task<bool> ReisterAndLoadUserDataAsync(RegisterModel registerModel);
         UserModel CurrentUser { get; }
+        Task LogoutAsync();
+        Task<bool> TryLoadUserFromDbAsync();
```

Comment:
Membership Service should not have Db related methods, It can use it inside, but outside must me just LoadUser or something like this